src/kernels/ipywidgets-message-coordination/ipyWidgetScriptSource.ts

```diff
@@ -55,7 +55,7 @@ export class IPyWidgetScriptSource {
         uriConverter.requestUri(
             (e) =>
                 this.postEmitter.fire({
-                    message: InteractiveWindowMessages.ConvertUriForUseInWebViewResponse,
+                    message: InteractiveWindowMessages.ConvertUriForUseInWebViewRequest,
```

Comment:
Possibly changed inadvertently as part of the web refactor.
This resulted in the Uri never getting resolved (basically a promise was never resolved & on the webview side the script was never made available)